spotify_tensorflow/tfx/tfdv.py

```diff
@@ -16,25 +16,62 @@
 # under the License.
 #
 
-import argparse
+import os
+import sys
+import time
 
-from apache_beam.options.pipeline_options import PipelineOptions
 import tensorflow_data_validation as tfdv
+from apache_beam.options.pipeline_options import GoogleCloudOptions, PipelineOptions, SetupOptions
+from spotify_tensorflow.tfx.utils import create_setup_file, assert_not_empty_string
+from tensorflow_metadata.proto.v0 import statistics_pb2  # noqa: F401
 
 
-class TFDV(object):
+class GenerateStats(object):
+    def __init__(self, input_data, output_path=None):
+        self.input_data = input_data
+        if output_path is None:
+            self.output_path = os.path.join(self.input_data, "_stats.pb")
```

Comment:
(minor) - could we rename `_stats.pb` to `stats.pb`?

Reply:
sure! probably we also need the similar change in mlpr.